pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -139,8 +139,12 @@ message LinuxPodSandboxConfig {
 message PodSandboxConfig {
     // The name of the sandbox.
     optional string name = 1;
+    // The uid of the sandbox.
```

Comment:
I think if we add a comment to the `CreatePodSandboxRequest` type saying: "If two requests have the same PodSandboxConfig.uid value, the server SHOULD return the same pod_sandbox_id value", that resolves my idempotency comment for the pod side of things... And, well, I'm totally for using natural idempotency tokens, which we have here.

Container still needs one, which I'll have a pr for shortly...